Assets/Scripts/Actors/ActorComponents/PlayerActorPhysics.cs

```diff
@@ -96,30 +117,33 @@ void JumpMovement()
 		{
 			currStoppingPower = stoppingSpeed;
 
-			CheckGroundSlope();
-
-			moveVec = inputVec * jumpMoveSpeed * groundSlopeSpeedMod;
+			moveVec = inputVec * jumpMoveSpeed;
 			moveVec.y = rigidbody.velocity.y;
 
-			lastVelocity = moveVec;
 			rigidbody.velocity = moveVec;
 
-			if ( actor.animator )
+			if ( _actor.animator )
```

Comment:
Other spots have _actor.animator != null, I prefer this shorter way of doing it. Either way pick one and be consistent.